qemu/README.md

```diff
@@ -0,0 +1,14 @@
+# QEMU Manager
```

Comment:
There is an issue to add packer templates #242 
Instruction to build such qcow2 should part of that issue